cli/src/test/kotlin/uk/gov/justice/digital/client/BlockingDomainClientTest.kt

```diff
@@ -73,8 +73,9 @@ class BlockingDomainClientTest {
     private fun createServerForScenario(scenario: String): EmbeddedServer {
         // Create an embedded server configured with the controller for the specified test scenario.
         val serverInstance = ApplicationContext.run(EmbeddedServer::class.java, mapOf(TEST_SCENARIO to scenario))
+        // Ensure the client is correctly configured
+        System.setProperty("http.client.url", "http://localhost:${serverInstance.port}")
```

Comment:
can we make "localhost" as variable similar to port?

Reply:
This isn't needed in the test. It's unlikely to be an issue in practise.